### DISCUSSION TITLE
Next js support

### DISCUSSION THREAD

**Discussion by abdulrahimiliasu** [khmyznikov/pwa-install#112] [Q&A] [answered]
Does this package support `Next.js` 13 or newer, if yes can you provide an example ?

**Comment by Siddesh0002T** [comment]
https://github.com/RubbySoft/fixmate/issues/6

**Reply by 7IBO** [reply]
Hi, this repo does not exist anymore. Could you please share the code if you do have it? Else did you remember what changes have you done to the legacy react file?

**Comment by khmyznikov** [comment, marked as answer]
- [React 19+ sample](https://stackblitz.com/edit/react-19-web-components-shoelace-8rdjcfbb?file=src%2FApp.jsx)
- [Next.js 15 + React 19 sample](https://stackblitz.com/edit/github-xrnbtug5?file=src%2Fapp%2F(delete-this-and-modify-page.tsx)%2FPWAInstall.tsx)
